### ISSUE TITLE
Active theme not ticked

### ISSUE THREAD

**Issue by stuaxo** [Cybolic/vineyard#12]
I've got the Ambiance look and feel installed with "Light" theme selected.

When vinyard lists "Light" but it's not ticked.

Screenshot run grabbed after running terminal from vinyard and running winecfg in it

https://imgur.com/a/mcj5n

**Comment by Cybolic** [comment]
Yeah, the theme supports is incredibly unfinished in the GUI. It should work through the Python library, but I never got around to finishing it in vineyard-preferences.
Would you happen to have any other themes I can test with, apart from "Light", so I might whip this back in shape?

**Comment by Cybolic** [comment]
The latest update should let you activate a theme. Note that I only have one to test with though, so it doesn't handle multiple themes very gracefully right now.

**Comment by stuaxo** [comment]
I'll see what I can find, definitely have a couple of (really old) wine prefixes with themes installed in them.   Real life getting in the way a bit,  so ping on here if nothing happens for a few days.
